tests/sentry/event_manager/test_event_manager.py

```diff
@@ -43,7 +43,7 @@
 
 def make_event(**kwargs):
     result = {
-        "event_id": "a" * 32,
+        "event_id": uuid.uuid1().hex,
```

Comment:
why this change ?

Reply:
We actually do want different events rather than duplicates in these tests

Reply:
you can pass the event_id as parameter. Anyway, if this is not breaking the other tests in the file, that's ok